cmd/csaf_downloader/main.go

```diff
@@ -10,21 +10,26 @@
 package main
 
 import (
+	"context"
 	"fmt"
 	"log"
 	"net/http"
 	"os"
+	"os/signal"
 
 	"github.com/csaf-poc/csaf_distribution/util"
 	"github.com/jessevdk/go-flags"
 )
 
+const defaultWorker = 1
```

Comment:
Shouldn't we use a default with at least 2 to demonstrate this advantage (and get more testing of it)?